### PR TITLE
test: add tests for root file

### PR DESCRIPTION
**Pull request by rajat-ws**
- added tests for `root` file under `services` folder

### CODE REVIEW

File: services/tests/root.test.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import MockAdapter from "axios-mock-adapter";
+import { getApiClient } from "../../utils/apiUtils";
+import { getRecommendations } from "../root";
+
+describe("root tests", () => {
+  it("should ensure it returns data when response is ok", async () => {
```
Comment (ihatelactose) [inline services/tests/root.test.ts]:
```suggestion
  it("should return data as expected", async () => {
```

File: services/tests/root.test.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    const mock = new MockAdapter(getApiClient().axiosInstance);
+    const resData = { results: [{ trackId: 12345, trackName: "Baarishein" }] };
+    mock.onGet(`/search?term=anuv`).reply(200, resData);
+    const res = await getRecommendations();
+    expect(res).toEqual(resData.results);
+  });
+
+  it("should ensure it returns an empty array when err coming in response ", async () => {
```
Comment (ihatelactose) [inline services/tests/root.test.ts]:
```suggestion
  it("should return an empty array if error is present in the response", async () => {
```

**Review by ihatelactose — APPROVED** [review]
Aside from the wordy noise this is good!

File: services/tests/root.test.ts (hunk 1 above)
Comment (rajat-ws) [reply services/tests/root.test.ts]: Updated accordingly.

File: services/tests/root.test.ts (hunk 2 above)
Comment (rajat-ws) [reply services/tests/root.test.ts]: Thanks for the review. Updated accordingly.

**Pull request merged**